src/main/scala/edu/ie3/powerFactory2psdm/config/model/WecConversionConfig.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.powerFactory2psdm.config.model
+
+import edu.ie3.powerFactory2psdm.config.ConversionConfigUtils.QCharacteristic
+import edu.ie3.powerFactory2psdm.config.model.IndividualModelConfig.ModelConversionMode
+import edu.ie3.powerFactory2psdm.config.model.WecConversionConfig.{
+  IndividualWecConfig,
+  WecModelConversionMode
+}
+import edu.ie3.powerFactory2psdm.generator.ParameterSamplingMethod
+
+/** General configuration for wec model conversion of static generators of
+  * category "Wind". As the model does not include most of the model parameters
+  * we can choose by setting the [[conversionMode]] to either convert the models
+  * to [[edu.ie3.datamodel.models.input.system.FixedFeedInInput]] or generate a
+  * [[edu.ie3.datamodel.models.input.system.WecInput]] by sampling the missing
+  * parameters.
+  *
+  * @param conversionMode
+  *   convert to fixed feed-in or generate model
+  * @param individualConfigs
+  *   for certain generators
+  */
+final case class WecConversionConfig(
+    conversionMode: WecModelConversionMode,
+    individualConfigs: Option[List[IndividualWecConfig]]
```

Comment:
are u 100% sure there will ever be a use case for this? Even though some particular single unit modification is required wouldn't it be easier to do this manually afterwards instead of introducing this as a feature for the converter? Bc this makes the converter more complex and error prone ...